src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0.45em;
+  margin-bottom: 0.4em;
```

Comment:
Why we need here this `margin`?

Reply:
Because there was a trouble with passing for one test - matching screenshots of pages. there were the text removed to bottom in my solution and it didn't match for source (I don't know why), and I find, that such values of margin h1 tag move all the text to necessary position. Should I have done it in a different way?